Model/src/main/java/org/gusdb/wdk/model/user/dataset/event/UserDatasetEventHandler.java

```diff
@@ -94,10 +94,10 @@ public UserDatasetEventHandler(
     this.projectId   = projectId;
     this.modelConfig = model;
 
-    this.eventRepo   = new UserDatasetEventRepo(dsSchema, ds);
-    this.installRepo = new InstalledUserDatasetRepo(dsSchema, ds);
-    this.shareRepo   = new UserDatasetShareRepo(dsSchema, ds);
-    this.ownerRepo   = new UserDatasetOwnerRepo(dsSchema, ds);
+    this.eventRepo   = new UserDatasetEventDBActions(dsSchema, ds);
```

Comment:
@Foxcapades  I've just said it under Steve's comment about the name but this looks like a good place to say it too, I'd call these a Dao. I don't care for the name but I think that's the name of a pattern you're using here, and I remember Daos in codebases using the Spring framework. Or like in https://www.baeldung.com/java-dao-pattern.